app/services/bgs/people_service.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module BGS
+  class PeopleService
+    include SentryLogging
+    class VaFileNumberNotFound < StandardError; end
+
+    def initialize(current_user)
+      @current_user = current_user
+    end
+
+    def find_person_by_ptcpnt_id
```

Comment:
why abbreviate `ptcpnt` vs. `participant`?

Reply:
Good point. I have changed it to participant